src/main/scala/com/kinja/soy/SoyWrites.scala

```diff
@@ -75,6 +75,13 @@ trait DefaultSoyWrites {
   }
 
   /**
+   * Converter for Float types.
+   */
+  implicit object FloatSoy extends SoyWrites[Float] {
```

Comment:
This was removed in https://github.com/gawkermedia/soy/commit/46cb078729a8b9bb07f4a3dfff4d354206d49161 because Google removed native support for Floats in the latest version. Why are you adding it back?